packages/kit/src/exports/public.d.ts

```diff
@@ -239,6 +239,17 @@ export interface Cookies {
 		opts: import('cookie').CookieSerializeOptions & { path: string }
 	) => void;
 
+	/**
+	 * Sets a cookie from a string. This will add a `set-cookie` header to the response, but also make the cookie available via `cookies.get` or `cookies.getAll` during the current request.
```

Comment:
```suggestion
	 * Sets a cookie from a string representing the value of the `set-cookie` header. This will add the `set-cookie` header to the response, and also make the cookie available via `cookies.get` or `cookies.getAll` during the current request.
```